.github/workflows/mtags-auto-release.yml

```diff
@@ -9,14 +9,18 @@ on:
         description: "Metals Version"
         required: true
         default: "v0.11.1"
+      metals_ref:
+        description: "Tag/branch-name from which run release"
+        required: true
+        default: "v0.11.1_mtags_release"
```

Comment:
We need to keep this at least until the next release.
Then we can remove this/or update it to next tag.